src/shared/components/publications-and-talks-section/videos-list/index.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import ReactPlayer from 'react-player';
+import PropTypes from 'prop-types';
+
+import RemainingVideo from 'shared/components/publications-and-talks-section/videos-list/remaining-video';
+import styles from './index.module.css';
+
+class PublicationsAndTalksList extends Component {
+    constructor(props) {
+        super(props);
+        const { videosArr } = this.props;
+        const remainingVideos = [];
+        let activeVideo;
+
+        videosArr.forEach((video) => {
```

Comment:
No need to iterate through all the videos, just use `find()` to retrieve the index of the active video.